src/main/java/org/embulk/output/kintone/KintonePageOutput.java

```diff
@@ -189,15 +144,14 @@ public void onRetry(
                 }
 
                 @Override
-                public void onGiveup(Exception firstException, Exception lastException)
-                    throws RetryGiveupException {}
+                public void onGiveup(Exception firstException, Exception lastException) {}
```

Comment:
Because IntelliJ IDEA gives the following warning.
```
Exception 'org.embulk.spi.util.RetryExecutor.RetryGiveupException' is never thrown in the method
```